CRM/Group/Form/Edit.php

```diff
@@ -84,7 +84,7 @@ class CRM_Group_Form_Edit extends CRM_Core_Form {
    * Set up variables to build the form.
    *
    * @return void
-   * @acess protected
+   * @access protected
```

Comment:
This can be simply removed. @access is an old php4 annotation that we're no longer using.